.github/workflows/build.yml

```diff
@@ -283,6 +287,7 @@ jobs:
       shell: bash
       run: |
         cd build
+        rm -Rf session-shared-build
```

Comment:
@Naatan It looks like you forgot to run `state run build-workflows` after a change.  So, this change ended up here...